pyanaconda/modules/storage/devicetree/viewer.py

```diff
@@ -527,12 +527,13 @@ def _get_windows_data(self):
                 continue
 
             device = self._get_device(blivet_device.name)
-            if str(device.part_type_uuid) == EFI_PARTITION_TYPE:
-                efi_partition = device
-                continue
+            if device and device.parted_partition:
```

Comment:
This is already solved in blivet with https://github.com/storaged-project/blivet/pull/1267/commits/c50e7a54c78dca853c27b116f8eee903be758652


This is not yet released though, that's why it still crashes. @vojtechtrefny when should we expect next blivet release? 

I dont object if you want to safe guard it on our side as well.

Reply:
Ah, we can close anaconda PR then.